x/auth/genaccounts/genesis_account.go

```diff
@@ -21,9 +23,13 @@ type GenesisAccount struct {
 	DelegatedVesting sdk.Coins `json:"delegated_vesting"` // delegated vesting coins at time of delegation
 	StartTime        int64     `json:"start_time"`        // vesting start time (UNIX Epoch time)
 	EndTime          int64     `json:"end_time"`          // vesting end time (UNIX Epoch time)
+
+	// module account fields
+	ModuleName string `json:"module_name"` // name of the module account
+	IsMinter   bool   `json:"is_minter"`   // used to differentiate ModuleMinterAccount from a ModuleHolderAccount
```

Comment:
Is this the API we want here? Should `IsMinter bool` be something like `ModulePath string`? (and we would need a router)

Reply:
I'm not following what are you referring exactly with `ModulePath`